ios/RNInstabug/InstabugReactBridge.h

```diff
@@ -58,7 +58,7 @@
 
 - (void)logVerbose:(NSString *)log;
 
-- (void)setReproStepsConfig: (IBGUserStepsMode)bugMode :(IBGUserStepsMode)crashMode:(IBGUserStepsMode)sessionReplayMode  :(IBGUserStepsMode)anr :(IBGUserStepsMode)appHangsMode :(IBGUserStepsMode)fatalCrashMode :(IBGUserStepsMode)nonFatalCrashMode :(IBGUserStepsMode)forceRestartMode : (IBGUserStepsMode) oomMode;
+- (void)setReproStepsConfig: (IBGUserStepsMode)bugMode :(IBGUserStepsMode)sessionReplayMode  :(IBGUserStepsMode) anr :(IBGUserStepsMode)appHangsMode :(IBGUserStepsMode)fatalCrashMode :(IBGUserStepsMode)nonFatalCrashMode :(IBGUserStepsMode)forceRestartMode : (IBGUserStepsMode) oomMode ;
```

Comment:
I think it'd be more readable if we put each parameter on a separate line and give them names rather than having all of them unnamed.